astropy/utils/tests/test_data.py

```diff
@@ -1001,6 +1001,7 @@ def test_is_url_in_cache_local(temp_cache, valid_urls, invalid_urls):
     assert not is_url_in_cache(nonexistent)
 
 
+@pytest.mark.filterwarnings('ignore:Not clearing data cache - cache inaccessible due to FileNotFoundError')
```

Comment:
@aarchiba , seems like #9575 didn't cache this one. Is handling it like this okay?

Reply:
I get no warnings on my machine, at least, though it's not tested in CI. No reason this should emit a warning - clear_download_cache should work fine (no warnings) on a nonexistent cache, and it shouldn't be running on a nonexistent cache here anyway. But if you can produce a warning, let me know and I'll try to track it down?